lib/datasource/remote/model/delegator_model.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:equatable/equatable.dart';
+
```

Comment:
Do you need equatable for this model?

Reply:
I try to use it for something and forgot to remove it. Thank you 👍  done